### ISSUE TITLE
Crashes on start()

### ISSUE THREAD

**Issue by koushik-boinepally** [CasperPas/flutter-sound-stream#64]
```
final RecorderStream _recorder = RecorderStream();
await _recorder.start();
```

Crashes instantly on iOS 15:

```
sound_stream/SwiftSoundStreamPlugin.swift:199: Fatal error: Unexpectedly found nil while unwrapping an Optional value
* thread #1, queue = 'com.apple.main-thread', stop reason = Fatal error: Unexpectedly found nil while unwrapping an Optional value
    frame #0: 0x00000001b47dca2c libswiftCore.dylib`_swift_runtime_on_report
libswiftCore.dylib`:
->  0x1b47dca2c <+0>: ret
libswiftCore.dylib`:
    0x1b47dca30 <+0>: b      0x1b47dca2c               ; _swift_runtime_on_report
libswiftCore.dylib`:
    0x1b47dca34 <+0>: adrp   x8, 366465
    0x1b47dca38 <+4>: ldrb   w0, [x8, #0x994]
Target 0: (Runner) stopped.
Lost connection to device.
```

#36 Solution mentioned here of changing podfile doesn't seem to work

**Comment by artyomkonyaev** [comment]
Hello @koushik-boinepally,

I am currently facing the same issue. Have you been able to solve it?

Thank you.

**Comment by koushik-boinepally** [comment]
Hey @artyomkonyaev 
I had to use a different library: [Mic Stream](https://pub.dev/packages/mic_stream) which did the job for me without issues as of now. I would contribute to this repo but I'm not well versed with anything sound related.

**Comment by CasperPas** [comment]
somehow `mRecordFormat` was not initialized. Perhaps, it was failed in the function `initializeRecorder`. I believe it was fixed in `v0.4.0`
